client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -49,7 +48,10 @@ const EditorSharingAccordion = React.createClass( {
 		const targeted = connections.filter( ( connection ) => {
 			return ! includes( skipped, connection.keyring_connection_ID );
 		} );
-		const targetedServices = serviceConnections.getServicesFromConnections( targeted );
+		const targetedServices = uniq( targeted.map( ( connection ) => ( {
```

Comment:
I believe plain `uniq` is not what you want here:

https://lodash.com/docs/4.16.4#uniq

Instead, `uniqBy` is the one that lets you specify an iteratee:

https://lodash.com/docs/4.16.4#uniqBy